paasta_tools/utils.py

```diff
@@ -2148,6 +2153,15 @@ def get_default_spark_iam_user(self) -> str:
             "default_spark_iam_user", "/etc/boto_cfg/mrjob.yaml"
         )
 
+    def get_spark_driver_default_pool_override(self) -> str:
+        """Get the spark driver's default pool override variable defined in this host's cluster config file.
```

Comment:
```suggestion
        """
        If defined, fetches the override for what pool to run a Spark driver in. Otherwise, returns the default Spark driver pool.
```

I think we usually have the docstring start with a newline - also, default pool tends to mean something specific at yelp (i.e., a spot pool where disruption is allowed), so I tweaked the wording a bit to avoid that :)